### PR TITLE
[DOC-103] ODS/API doc reformatting

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: docs/reference/2-ods-api/platform-dev-guide/fundamentals.md [hunk 1]
```diff
@@ -204,18 +204,3 @@ with every code commit into the version control system. This practice provides
...
-Dashboards display a range of student performance metrics in a compelling
-interface. The ODS database contains the full spectrum of information needed to
-populate the Dashboards, and the data structures in the ODS are tuned to the
-data loaders that ship with the Dashboards. This similarity allows for a
-simplified data import and metrics computation in organizations where the Ed-Fi
-Dashboards are used.
-
-1 See the [Ed-Fi Technical Suite Compatibility
-Matrix](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875825/Ed-Fi+Technical+Suite+Version+Matrix) for
-the latest details on the Ed-Fi products that are compatible with the ODS / API.
```
Comment (stephenfuqua) [inline docs/reference/2-ods-api/platform-dev-guide/fundamentals.md:221]: !!!!

File: docs/reference/2-ods-api/how-to-guides/how-to-add-api-composites-to-the-ed-fi-ods-api-solution.md [hunk 2]
```diff
@@ -124,10 +124,7 @@ of a composite URL.
 :::note
 
 The following GitHub link contains source files for the Composite described in
-this article:
-
-* [ODS/API 7.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.2/Samples/Composites)
-* [ODS/API 7.1](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.1/Samples/Composites)
-* [ODS/API 6.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v6.2/Samples/Composites)
+this article: [ODS/API
+7.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.2/Samples/Composites)
```
Comment (stephenfuqua) [inline docs/reference/2-ods-api/how-to-guides/how-to-add-api-composites-to-the-ed-fi-ods-api-solution.md:128]: I went ahead and simplified to list only the current version here, since we are going forward with the full versioning process.

At some point we may bring in the 7.1 and 6.2 documents as well, but probably not until next year. When we do that, I'm thinking of starting from the 7.2 pages. Separately, we can use WinMerge to find the differences between the two exports from Confluence (for example, comparing the 7.2 and 7.1 exports), which will make it easy to remove anything from 7.2 that does not belong in 7.1

In other words, I don't want to go through all of this migration work again from the raw 7.1 or 6.2 exported files. Use the 7.2 markdown files as the basis and adjust them as needed.

File: docs/reference/2-ods-api/how-to-guides/how-to-clear-expired-client-access-tokens-with-a-database-job.mdx
Comment (stephenfuqua) [inline docs/reference/2-ods-api/how-to-guides/how-to-clear-expired-client-access-tokens-with-a-database-job.mdx:1]: I renamed the file above this but accidentally did it in a way that git does not recognize it as a rename. I did not change anything significant about the content - just fixed some style issues and broken links, and introduced tabs for MSSQL / PostgreSQL differentiation.

File: docs/reference/2-ods-api/how-to-guides/how-to-enable-logging-api-request-and-response-content.mdx
Comment (stephenfuqua) [inline docs/reference/2-ods-api/how-to-guides/how-to-enable-logging-api-request-and-response-content.mdx:1]: Another example of a file that was renamed to `.mdx` so that I could introduce a tabbed display.

File: docs/reference/2-ods-api/platform-dev-guide/readme.md [hunk 3]
```diff
@@ -27,7 +27,3 @@ familiar with the following knowledge areas:
     see Chapter 5
 * [OAuth 2.0](https://tools.ietf.org/html/rfc6749)
 * [JavaScript Object Notation](http://www.json.org) (JSON)
-
-## Developers' Guide Contents
-
-Find out more about how to develop platforms based on the Ed-Fi ODS / API:
```
Comment (stephenfuqua) [inline docs/reference/2-ods-api/platform-dev-guide/readme.md:33]: Another place where I could add a child page display if you want it here.

File: docs/reference/2-ods-api/whats-new/whats-new-in-prev-v7x-releases.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+---
+description: Summary of what changed in prior releases of 7.x
+---
```
Comment (stephenfuqua) [inline docs/reference/2-ods-api/whats-new/whats-new-in-prev-v7x-releases.md:3]: Adding a description like this improves the display of the DocCardList in `readme.md` - in other words, the display of child pages.

image

File: docs/reference/2-ods-api/how-to-guides/how-to-load-the-ods-with-sample-xml-data-using-bulk-load-client-utility.mdx [hunk 5]
```diff
@@ -0,0 +1,460 @@
...
+* Restart the instance of the Ed-Fi ODS / API running locally again to refresh
+  the claims.
+* Open a PowerShell session.
+* Navigate to `Ed-Fi-SampleDataLoad` directory (e.g.,
+  `C:\Ed-Fi-SampleDataLoad`).
+* Open the `LoadSampleData.ps1` file in notepad or Windows PowerShell ISE
+  and update the $apiLoaderExe variable Value  where
+  `EdFi.BulkLoadClient.Console` is installed in **Step 1.**
+* Example `$apiLoaderExe="<YourInstallFolderFullPath>\EdFi.BulkLoadClient.Console.exe"`
+* For Postgres Only, please update Key &amp; Secret `"-k", "**E**mpty", "-s", "empty**S**ecret"`
```
Comment (vimayya) [inline docs/reference/2-ods-api/how-to-guides/how-to-load-the-ods-with-sample-xml-data-using-bulk-load-client-utility.mdx:443]:
```suggestion
* For Postgres Only, please update Key &amp; Secret `"-k", "Empty", "-s", "emptySecret"`
```

**Review by vimayya — APPROVED** [review]

File: docs/reference/2-ods-api/how-to-guides/how-to-load-the-ods-with-sample-xml-data-using-bulk-load-client-utility.mdx (hunk 5 above)
Comment (stephenfuqua) [reply docs/reference/2-ods-api/how-to-guides/how-to-load-the-ods-with-sample-xml-data-using-bulk-load-client-utility.mdx:443]: I thought the `*` were on purpose 😆
